spec/tsuru_helper.rb

```diff
@@ -0,0 +1,116 @@
+require 'tempfile'
+require 'fileutils'
+require 'command_line_helper'
+
+require 'rspec/retry'
+
+RSpec.configure do |c|
+  c.add_setting :debug_commands, :default => false
+
+  c.add_setting :deploy_env, :default => ENV['DEPLOY_ENV'] || 'ci'
+
+  c.add_setting :target_platform, :default => ENV['TARGET_PLATFORM'] || 'aws'
+
+  # If nil will be populated based on the default platform
+  c.add_setting :target_api_host, :default =>
+    case c.target_platform
+    when 'aws'
+      "#{RSpec.configuration.deploy_env}-api.tsuru.paas.alphagov.co.uk"
+    when 'gce'
+      "#{RSpec.configuration.deploy_env}-api.tsuru2.paas.alphagov.co.uk"
```

Comment:
Is there a way we can take the domain_name parameter from platform-aws.yml and platform-gce.yml respectively rather than having them hardcoded in here?